src/main/java/in/gov/cdc/cdcdemostage/validators/dob/DobObjectValidator.java

```diff
@@ -0,0 +1,142 @@
+package in.gov.cdc.cdcdemostage.validators.dob;
+
+
+import in.gov.cdc.cdcdemostage.models.Event;
+import in.gov.cdc.cdcdemostage.models.ExtractedPacket;
+import in.gov.cdc.cdcdemostage.models.ValidationError;
+import in.gov.cdc.cdcdemostage.validators.specs.*;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.BitSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * This class will orchestrate Dob validation checks against the incoming request.
+ * Outcome of the validation would be recorded in the reject bitmap
+ * indicator along with the detailed error code.
+ */
+
+@Component
+public class DobObjectValidator implements IObjectValidator {
+
+
+
+
+    // Constructor injection of all validators
+    List<IValidator> validators;
+
+
+    @Override
+    public String mode(String mode) {
+        return null;
+    }
+
+    @Override
+    public String requestType(String requestType) {
```

Comment:
requestType can be extracted from Event. This might not be required on the interface

Reply:
Agree